### ISSUE TITLE
Traffic Graph

### ISSUE THREAD

**Issue by Unrestricted-User** [safing/portmaster#698]
**What would you like to add or change?**:
Add a graph like in [Glass Wire](https://www.glasswire.com/) to display and monitor what applications are using your data and to **see** suspicious activity with your own eyes

**Why do you and others need this?**:
Because it looks nice, and it would be a "nice to have" sort of thing
it would also allow you to see if any allowed applications are maybe not as trustworthy as you might have thought

[![GlassWire](https://user-images.githubusercontent.com/57331183/174095553-e6152dc3-9cd2-47cd-beb3-b4e14c48623b.png)
](https://drive.google.com/uc?export=download&id=1HLSJf6Vf9uGtcGIXwhf7kdA0O5yLkLDF)

**Comment by JS-E** [comment]
I do think this would be an awesome feature to have. It's a very useful tool to see how certain applications are potentially hogging the network utilization. Would be cool if these could be logged too, so you could see the history of how applications use it over time. Great idea 👍

**Comment by davegson** [comment]
related discussion #151

we do like the idea of graphs - but there are a few bigger things that need to happen first before we could attempt this.

- Namely the Full-Device Network Activity (https://github.com/safing/portmaster/issues/683), which is currently being worked on. 
- And then the History Module (https://github.com/safing/portmaster/issues/682, https://github.com/safing/portmaster-ui/issues/226) to allow viewing a longer timeframe than the last 10 minutes.

So putting it into the Backlog for now.

**Comment by Raphty** [comment]
I am cleaning out old issues. If you feel this issue should not have been closed let me know.

Please keep in mind, the free version of Portmaster only has limited support.
For free users our active Discord community as well as the chat bot are the fastest and best way to get their help. https://discord.gg/safing
If you find our work brings value to you, please consider supporting it by purchasing Plus or Pro Packages https://safing.io/pricing/.
If you are already a subscriber, first Thank You! and also if you want priority support pleas send in an email and let me know your username so I can prioritize your request accordingly.
